src/services/Viewer.js

```diff
@@ -187,6 +199,7 @@ export default class Viewer {
 		this._state.onNext = onNext
 		this._state.onClose = onClose
 		this._state.canLoop = canLoop
```

Comment:
All those 
```js
		this._state.files = list
		this._state.loadMore = loadMore
		this._state.onPrev = onPrev
		this._state.onNext = onNext
		this._state.onClose = onClose
		this._state.canLoop = canLoop
```

should not be set if `this.el` is not null, as we don't support opening multiple files in a single-display view